galleries/examples/lines_bars_and_markers/stackplot_demo.py

```diff
@@ -20,11 +20,11 @@
 # https://population.un.org/wpp/, license: CC BY 3.0 IGO
 year = [1950, 1960, 1970, 1980, 1990, 2000, 2010, 2018]
 population_by_continent = {
-    'africa': [228, 284, 365, 477, 631, 814, 1044, 1275],
-    'americas': [340, 425, 519, 619, 727, 840, 943, 1006],
-    'asia': [1394, 1686, 2120, 2625, 3202, 3714, 4169, 4560],
-    'europe': [220, 253, 276, 295, 310, 303, 294, 293],
-    'oceania': [12, 15, 19, 22, 26, 31, 36, 39],
+    'africa': np.array([228, 284, 365, 477, 631, 814, 1044, 1275])*10e6,
```

Comment:
```suggestion
    'africa': np.array([228, 284, 365, 477, 631, 814, 1044, 1275])*1e6,
```

If these are "millions", then the constant is 1e6 rather than 10e6